packages/apps/src/SideBar/index.tsx

```diff
@@ -65,31 +52,24 @@ class SideBar extends React.PureComponent<Props, State> {
     );
   }
 
-  private collapse = (): void => {
-    this.setState(({ isCollapsed }: State) => ({
-      isCollapsed: !isCollapsed
-    }), () => {
-      store.set('sidebar', this.state);
-    });
-  }
-
   private renderCollapse () {
-    const { isCollapsed } = this.state;
+    const { isCollapsed } = this.props;
 
     return (
       <div className='apps--SideBar-collapse'>
         <Button
-          icon={`angle double ${isCollapsed ? 'right' : 'left'}`}
+          icon={`angle double right`}
```

Comment:
Small nitpick - This works with a simple `icon='angle double right'` as well (no template string indicators and no unneeded `{}'s`

Reply:
Amended